src/SuperDumpService/wwwroot/js/ModalPopupWindow.js

```diff
@@ -0,0 +1,294 @@
+﻿var bailwal_modalPopupWindow = null;
```

Comment:
please add license and source information for this code. does the license permit usage?